client/js/components/drawer.js

```diff
@@ -0,0 +1,10 @@
+import showHide from './show-hide';
+
+export default (el) => {
+  const toggle = showHide({
+    el,
+    activeClass: 'is-drawer-open'
+  });
+
+  toggle.trigger.addEventListener('click', toggle.toggleActive);
```

Comment:
We use this a few times, what about having a `js-show-hide` that sets has a generic show-hide class?